### ISSUE TITLE
103.0 will break this componment

### ISSUE THREAD

**Issue by froz** [snarky-snark/home-assistant-variables#13]
[Do not select all entities when omitting entity ID in service call](https://github.com/home-assistant/home-assistant/commit/02d9ed5e36257490726a75c8440b4a2b61bc7fe4#)

**Comment by snarky-snark** [comment]
The fix, it seems, is to replace references to `ENTITY_SERVICE_SCHEMA` with `make_entity_service_schema`.

Users may need to update any calls to `var` services that previously omitted `entity_id` to imply all entities.

Next release will not be backward compatible. HACS manifest should be updated accordingly, and a note added to the release.

**Comment by Swiftnesses** [comment]
Ouch, this really caught me off guard, I'll try and hotfix my install until an official release. Thanks for the fix!

**Comment by Swiftnesses** [comment]
Tried the above suggestion, no luck my end.

```
Setup failed for var: unknown error
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/setup.py", line 138, in _async_setup_component
    component = integration.get_component()
  File "/usr/src/homeassistant/homeassistant/loader.py", line 223, in get_component
    cache[self.domain] = importlib.import_module(self.pkg_path)
  File "/usr/local/lib/python3.7/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
  File "<frozen importlib._bootstrap>", line 983, in _find_and_load
  File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 728, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/config/custom_components/var/__init__.py", line 59, in <module>
    SERVICE_SET_SCHEMA = make_entity_service_schema.extend({
AttributeError: 'function' object has no attribute 'extend'
```

**Comment by snarky-snark** [comment]
@Swiftnesses, you are running 0.103.0?

Replace `ENTITY_SERVICE_SCHEMA.extend({` with `make_entity_service_schema({`

Otherwise, replace `ENTITY_SERVICE_SCHEMA` with `make_entity_service_schema` verbatim.

**Comment by Swiftnesses** [comment]
Sorted for now, thanks :)

**Comment by snarky-snark** [comment]
Fixed in [v0.9.0](https://github.com/snarky-snark/home-assistant-variables/releases/tag/v0.9.0).
